sdks/python/container/base_image_requirements.txt

```diff
@@ -57,6 +57,7 @@ mmh3==2.5.1
 # These are additional packages likely to be used by customers.
 numpy==1.19.5
 scipy==1.4.1
+scikit-learn==0.24.1 ; python_version>="3.6"
```

Comment:
nit: do we need this version check? Isn't 3.6 the lowest we support?

Reply:
Removed. You are right, we don't need it.